plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -86,4 +97,36 @@ def stub_get_build_id_from_queue(build_id)
       jenkins.job_url(96).must_equal "https://jenkins.zende.sk/job/rdhanoa_test_project/96/"
     end
   end
+
+  describe "#build_with_env_flags" do
```

Comment:
don't see a build_with_env_flags method anywhere ... should be `"build with env flags"` ?